tools/benchmark.py

```diff
@@ -25,61 +33,81 @@ def main():
     args = parse_args()
 
     cfg = Config.fromfile(args.config)
+    if args.work_dir is not None:
+        mmcv.mkdir_or_exist(osp.abspath(args.work_dir))
+        timestamp = time.strftime('%Y%m%d_%H%M%S', time.localtime())
+        json_file = osp.join(args.work_dir, f'fps_{timestamp}.json')
```

Comment:
```suggestion
        json_file = osp.join(args.work_dir, args.config， f'fps_{timestamp}.json')
```

It is better to add the config name.